Rakefile

```diff
@@ -30,3 +30,99 @@ task :docstats, [:path] do |_, args|
 end
 
 Rake::ExtensionTask.new('drawText')
+
+desc 'Create a new version of the release-toolkit gem'
+task :new_release do
+  version_file = File.join('lib', 'fastlane', 'plugin', 'wpmreleasetoolkit', 'version.rb')
+  require_relative(version_file)
+  puts ">>> Current version is: #{Fastlane::Wpmreleasetoolkit::VERSION}"
+  puts ">>> Pending CHANGELOG:\n" + get_changelog_section(file: 'CHANGELOG.md', section_title: 'Develop').map { |l| "| #{l}" }.join
+
+  puts '>>> New version to use?'
+  new_version = STDIN.gets.chomp
+
+  ### VERSION constant
+  puts '>>> Updating `VERSION` constant in `version.rb`...'
+  content = File.read(version_file)
+  content.gsub!(/VERSION = .*/, "VERSION = '#{new_version}'")
+  File.write(version_file, content)
+  sh('bundle', 'install') # To update Gemfile.lock with new wpmreleasetoolkit version
+
+  ### CHANGELOG.md
+  puts '>>> Updating the `CHANGELOG.md` file...'
+  update_changelog_sections(
+    file: 'CHANGELOG.md',
+    wip_header_title: 'Develop',
+    placeholder_sections: ['Breaking Changes', 'New Features', 'Bug Fixes', 'Internal Changes'],
+    new_section_title: new_version
+  )
```

Comment:
Simply put what this method does is:
1. Replace the title of the `## Develop` section with `## x.y` version name
2. Get rid of any empty section (section that might have been left with just `_None_` and no item) to avoid listing them uselessly
3. Insert a new placeholder section (`## Develop` title + the placeholders for the 4 subsections) – to be ready for when we start working on the next version after that release